posthog/ai/openai/openai.py

```diff
@@ -29,14 +28,38 @@ def __init__(self, posthog_client: PostHogClient, **kwargs):
         """
         super().__init__(**kwargs)
         self._ph_client = posthog_client
-        self.chat = WrappedChat(self)
-        self.embeddings = WrappedEmbeddings(self)
-        self.beta = WrappedBeta(self)
-        self.responses = WrappedResponses(self)
 
+        # Store original objects after parent initialization (only if they exist)
+        self._original_chat = getattr(self, "chat", None)
+        self._original_embeddings = getattr(self, "embeddings", None)
+        self._original_beta = getattr(self, "beta", None)
+        self._original_responses = getattr(self, "responses", None)
 
-class WrappedResponses(openai.resources.responses.Responses):
-    _client: OpenAI
+        # Replace with wrapped versions (only if originals exist)
+        if self._original_chat is not None:
+            self.chat = WrappedChat(self, self._original_chat)
+
+        if self._original_embeddings is not None:
+            self.embeddings = WrappedEmbeddings(self, self._original_embeddings)
+
+        if self._original_beta is not None:
+            self.beta = WrappedBeta(self, self._original_beta)
+
+        # Only add responses if available (newer OpenAI versions)
```

Comment:
Useless comment, the above comment explains it

Reply:
fair. Thats the ai adding random comments that i didnt reject like i usually do